src/pretix/base/models/customers.py

```diff
@@ -45,6 +46,7 @@ class Customer(LoggedModel):
     organizer = models.ForeignKey(Organizer, related_name='customers', on_delete=models.CASCADE)
     identifier = models.CharField(max_length=190, db_index=True, unique=True)
     email = models.EmailField(db_index=True, null=True, blank=False, verbose_name=_('E-mail'), max_length=190)
+    phone = PhoneNumberField(null=True, blank=True, verbose_name=_('Phone number'))
```

Comment:
`anonymize()` should clear the phone number as well

Reply:
Thanks for pointing that out. I added it to `models.Customer.anonymize`